docker/ssh/Dockerfile

```diff
@@ -20,5 +20,11 @@ RUN chown kernelci: /home/kernelci
 # SSH server will run on PORT 22
 EXPOSE 22
 
+# Copy authorized_keys file
+COPY authorized_keys /home/kernelci/.ssh
```

Comment:
We might need to run `mkdir -p /home/kernelci/.ssh` first as I don't think the directory will be created automatically by `useradd`.  If we don't create it, then the `authorized_keys` file will be copied as a `.ssh` file.

Another approach is to add a volume for the `.ssh` directory with the `authorized_keys` file provided by the host, maybe that would be more flexible.  It means the container could be created even if there isn't an `authorized_keys` file as it can be added later while the container is running.

Reply:
I like the latter approach more as it makes the authorized_keys file persistent.